pyk/src/pyk/tests/test_kastManip.py

```diff
@@ -152,17 +152,15 @@ def test(self):
         self.assertEqual(config_actual, config_expected)
 
 
-class BooleanTest(TestCase):
+class SimplifyBoolTest(TestCase):
 
     def test_simplify_bool(self):
```

Comment:
```suggestion
    def test_simplify_bool(self) -> None:
```

Enable type checking here by adding a type signature.

Reply:
I will get this in follow-on PR.